write-fonts/src/tables/gvar.rs

```diff
@@ -254,11 +262,15 @@ impl GlyphVariations {
             .map(|tup| tup.build(shared_tuple_map, shared_points.as_ref()))
             .unzip();
 
-        GlyphVariationData {
+        let mut this = GlyphVariationData {
```

Comment:
Nit: the name "this" is perhaps suboptimal to readers with a C++ or Java background.